scripts/components/Notebook.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+import NoteActions from '../actions/NoteActions';
+
+export default class Notebook extends React.Component {
```

Comment:
considered making this a `function` instead of a `class` but will need to do event firing later